# A collection of the abcd coefficients of a fifth order polynomial.
# this output is produced by an optimizer (nlp or qp)
# x(t) = a*t^5 + b*t^4 + c*t^3 + d*t^2 + e*t + f

float64[6] coeff_x
float64[6] coeff_y
uint32 id
float64 duration

